qadi_simple.py

```
@@ -0,0 +1,342 @@
+
+# Override the questioning prompt with a simpler version
+class SimplerQADIOrchestrator(SimpleQADIOrchestrator):
+    """QADI orchestrator with simplified Phase 1."""
+    
+    def __init__(self, temperature_override: Optional[float] = None) -> None:
+        super().__init__(temperature_override)
+        # Use custom prompts
+        self.prompts = SimplerQADIPrompts()
+
+
+async def run_qadi_analysis(
```

Ruff (0.12.2)

`run_qadi_analysis` is too complex (28 > 10)

(C901)